platform/asterix/boot/src/drivers/display/resources/pebble_logo.xbm

```diff
@@ -1,35 +1,31 @@
 #define pebble_logo_width 105
```

Comment:
Using [this](https://gist.github.com/blockarchitech/e75fe42908270ac62330f7a72b787c70) little script I wrote, I discovered that the width should probably be bumped to 112?

Reply:
it displays fine on bootloader as is